bundles/io/org.eclipse.smarthome.io.rest.sitemap/src/main/java/org/eclipse/smarthome/io/rest/sitemap/internal/PageChangeListener.java

```diff
@@ -199,7 +200,10 @@ public void stateUpdated(Item item, State state) {
                 event.widgetId = itemUIRegistry.getWidgetId(w);
                 event.visibility = itemUIRegistry.getVisiblity(w);
                 // event.item contains data from the item including its state (in event.item.state)
-                event.item = EnrichedItemDTOMapper.map(item, false, null, null);
+                String widgetTypeName = w.eClass().getInstanceTypeName()
+                        .substring(w.eClass().getInstanceTypeName().lastIndexOf(".") + 1);
+                event.item = EnrichedItemDTOMapper.map(item, "mapview".equalsIgnoreCase(widgetTypeName),
```

Comment:
Could you extract `"mapview".equalsIgnoreCase(widgetTypeName)` into a variable for readability?

Reply:
Ok